extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/handler/Lib485BridgeHandler.java

```diff
@@ -61,7 +61,7 @@ protected void openConnection() {
                     try {
                         socket = new Socket(receiverNode.getAddressString(), receiverNode.getPort());
                     } catch (IOException e) {
-                        logger.debug("Could not connect to {} in {}: {}", receiverNode, this.thing.getUID(), e);
+                        logger.debug("Could not connect to {} in {}: ", receiverNode, this.thing.getUID(), e);
```

Comment:
since this is debug logging provide an `e.getMessage()` instead and re-add the placeholder.